bin/MadGraph5_aMCatNLO/cards/WZ_leptonic/WLLjj_dim8_ewk_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set complex_mass_scheme False
+import model SM_Ltotal_Ind5v2020v2_UFO
+
+define p = g u c d s u~ c~ d~ s~
+define j = p b b~
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+
+generate    p p > W+ Z j j QED=4 QCD=0 NP=1 @ 1
+add process p p > W- Z j j QED=4 QCD=0 NP=1 @ 2
+output WLLjj_dim8_ewk
```

Comment:
Better name would be "WZjj"  to keep things consistent with the WWjj
+ You only have Z contributions

Reply:
@sihyunjeon okay it's changed to WZjj